applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_interface_element.cpp

```diff
@@ -153,66 +153,11 @@ void UPwSmallStrainInterfaceElement<TDim, TNumNodes>::CalculateMassMatrix(Matrix
 {
     KRATOS_TRY
 
+    // Resizing mass matrix, no mass in these interface elements
     const unsigned int N_DOF = this->GetNumberOfDOF();
-
-    // Resizing mass matrix
-    if (rMassMatrix.size1() != N_DOF) rMassMatrix.resize(N_DOF, N_DOF, false);
+    rMassMatrix.resize(N_DOF, N_DOF, false);
     noalias(rMassMatrix) = ZeroMatrix(N_DOF, N_DOF);
```

Comment:
I guess that when we remove the `noalias` construct, we can simplify this further to:
```
    rMassMatrix = ZeroMatrix{this->GetNumberOfDOF(), this->GetNumberOfDOF()};
```

In that case, I would also suggest to remove the lines containing `KRATOS_TRY` and `KRATOS_CATCH`.